src/chronify/__init__.py

```diff
@@ -1,4 +1,18 @@
 import importlib.metadata as metadata
 
+from chronify.store import Store
```

Comment:
I'm proposing that we allow users to import all common classes in the chronify namespace.
```python
from chronify import Store
```

Reply:
Makes sense to me. We should follow this on infrasys as well